docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -31,6 +31,12 @@ following steps:
    API server, each with its own identity, as well as an additional
    kubeconfig file for administration named `admin.conf`.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
+   writes the kubelet init configuration into `/var/lib/kubelet/config/init/kubelet` file. 
+   see [Set Kubelet parameters via a config file](/docs/tasks/administer-cluster/kubelet-config-file.md) 
+   and [Reconfigure a Node's Kubelet in a Live Cluster](/docs/tasks/administer-cluster/reconfigure-kubelet.md) 
+   for more information about Dynamic Kubelet Configuration.
```

Comment:
... This functionality is now by default disabled as it is behind a feature gate, but is expected to be a default in future versions.